common/tools/dev-tool/src/commands/about.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license
+
+import chalk from "chalk";
+
+import { baseCommands, baseCommandInfo } from ".";
+import { resolveProject } from "../util/resolveProject";
+import { createPrinter } from "../util/printer";
+import { leafCommand, makeCommandInfo } from "../framework/command";
+import { printCommandUsage } from "../framework/printCommandUsage";
+
+const log = createPrinter("help");
+
+const banner = `\
+       _______        __                __              __
+      / / ___/   ____/ /__ _   __      / /_____  ____  / /
+ __  / /\\__ \\   / __  / _ \\ | / /_____/ __/ __ \\/ __ \\/ / 
+/ /_/ /___/ /  / /_/ /  __/ |/ /_____/ /_/ /_/ / /_/ / /  
+\\____//____/   \\__,_/\\___/|___/      \\__/\\____/\\____/_/   
+
+Developer quality-of-life command for the Azure SDK for JS
+`;
+
+export const commandInfo = makeCommandInfo("about", "display command help and information");
+
+export default leafCommand(commandInfo, async (options) => {
```

Comment:
Should we use TSDoc? I like the idea of having the editor show what things mean. I think TSDoc would help!